mods/ra/rules/defaults.yaml

```diff
@@ -1097,16 +1097,20 @@
 	GrantCondition@IDISABLE:
 		RequiresCondition: lowpower || powerdown
 		Condition: disabled
-	CanPowerDown:
-		PowerupSound: EnablePower
-		PowerdownSound: DisablePower
-		PowerdownCondition: powerdown
+	ToggleConditionOnOrder:
+		DisabledSound: EnablePower
+		EnabledSound: DisablePower
+		Condition: powerdown
+		OrderName: PowerDown
 	WithDecoration@POWERDOWN:
 		Image: poweroff
 		Sequence: offline
 		Palette: chrome
 		RequiresCondition: powerdown
 		ReferencePoint: Center
+	PowerMultiplier@POWERDOWN:
+		RequiresCondition: powerdown
+		Modifier: 0
```

Comment:
This makes several yaml entries rendundant (in all our mods with the power-down option), for example on `IRON` in RA:
```
	Power:
		RequiresCondition: !powerdown
```

Reply:
Done.